### PR TITLE
fix tmpfs recognition to skip readahead

### PR DESCRIPTION
**Pull request by HereThereBeDragons**
Issue  #3315 

Extends/fixes #3262

### CODE REVIEW

**Comment by HereThereBeDragons** [comment]
@cernvm-bot cloudtest

File: test/unittests/t_util.cc [hunk 1]
```diff
@@ -407,6 +407,8 @@ TEST_F(T_Util, GetFileSystemInfo) {
   EXPECT_EQ(kFsTypeProc, fs_info.type);
   fs_info = GetFileSystemInfo("/");
   EXPECT_EQ(kFsTypeUnknown, fs_info.type);
+  fs_info = GetFileSystemInfo("/run");
```
Comment (jblomer) [inline test/unittests/t_util.cc]: That only works on Linux, and perhaps not on all distributions.  I'd check if `/run` exists first.

**Review by jblomer — CHANGES_REQUESTED** [review]

File: test/unittests/t_util.cc (hunk 1 above)
Comment (HereThereBeDragons) [reply test/unittests/t_util.cc]: seeing that this also fails on ubuntu i guess we either find some better tmpfs that is always there or we just remove the test?

Comment (jblomer) [reply test/unittests/t_util.cc]: Probably best to remove it then.  It's a pity but it's tough to test it reliably without being root.

**Comment by HereThereBeDragons** [comment]
@cernvm-bot cloudtest

**Comment by jblomer** [comment]
@cernvm-bot unittest

**Review by jblomer — APPROVED** [review]

**Review by vvolkl — APPROVED** [review]

**Pull request merged**